packages/accounts-controller/package.json

```diff
@@ -76,6 +76,7 @@
   },
   "peerDependencies": {
     "@metamask/keyring-controller": "^19.0.0",
+    "@metamask/network-controller": "^22.1.1",
```

Comment:
Should we add this to `devDependencies` as well?

Reply:
Updated